gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -159,19 +161,41 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
 
       for (const auto & parameter : parameters) {
         std::string param_name = parameter.get_name();
+        bool is_slip_param = param_name.find("slip_compliance") != std::string::npos;
 
-        if (param_name.find("slip_compliance") != std::string::npos) {
-          double slip = parameter.as_double();
-          if (slip < 0.) {
+        if (is_slip_param || param_name.find("friction_coefficient") != std::string::npos)
```

Comment:
Maybe it's just because I haven't had my coffee yet, but IMO the code would be more readable if we kept the validation logic separate for slip and friction parameters. I.e. just duplicate this block for friction coefficient parameters.

Reply:
I was attempted to avoid code duplication but I agree that your suggestion is more readable 0ef6f4018a455e9164f43658a4b3d84fd06f8fc9